web/package.json

```diff
@@ -85,4 +86,4 @@
   "volta": {
     "node": "20.16.0"
   }
-}
+}
```

Comment:
revert this and add the newline back please

Reply:
Sorry for that...